lib/components/viewers/RouteRow.js

```diff
@@ -68,6 +106,31 @@ const SimpleRouteRenderer = styled.span`
   padding: 0;
 `
 
+const PatternButton = styled.button`
+  background: transparent;
+  border: none;
+  border-radius: 50%;
+  color: #004f96;
+  opacity: ${(props) => (props.display ? 100 : 0)};
+  height: 40px;
+  margin-right: 8px;
+  position: relative;
+  transition: all ease-out 0.17s;
+  min-width: 40px;
+  z-index: ${(props) => (props.display ? 2 : -2)};
+
+  svg {
+    margin-top: -3px;
+    height: 22px !important;
+    width: 22px !important;
+  }
+
+  &:hover {
```

Comment:
Can we add an `:active` state for this button as well? Maybe `#ddd` or something or actually `#eeeeeeaa` is usually a classic trick

Reply:
maybe just opacity: 0.8